pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -357,6 +359,25 @@ func (r *ReconcileComplianceScan) phaseRunningHandler(instance *compv1alpha1.Com
 					return reconcile.Result{}, err
 				}
 				return reconcile.Result{}, nil
+			} else if goerrors.As(err, &unschedulableErr) {
+				// Create custom error message for this pod that couldn't be scheduled
+				cmName := getConfigMapForNodeName(instance.Name, node.Name)
+				cm := utils.GetResultConfigMap(instance, cmName, "error-msg", node.Name,
+					[]byte(err.Error()), false, common.PodUnschedulableExitCode)
+				cmKey := types.NamespacedName{Name: cm.Name, Namespace: cm.Namespace}
+				foundcm := corev1.ConfigMap{}
+				cmGetErr := r.client.Get(context.TODO(), cmKey, &foundcm)
+				if errors.IsNotFound(cmGetErr); cmGetErr != nil {
```

Comment:
This condition and the following one sounds strange, wouldn't the first condition hit for any `cmGetErr != nil` meaning that the second condition is never hit? Don't you just mean to say `if errors.IsNotFound(cmGetErr)` ?